python/src/opendp/_convert.py

```diff
@@ -320,6 +326,26 @@ def _slice_to_string(raw: FfiSlicePtr) -> str:
     return value.decode()
 
 
+def _bitvector_to_slice(val: Sequence[Any]) -> FfiSlicePtr:
+    np = import_optional_dependency('numpy', raise_error=False)
+    if np is not None and isinstance(val, np.ndarray):
+        val = val.tobytes()
+    
+    if not isinstance(val, (bytes, bytearray)):
+        raise TypeError("Expected type is BitVector but input data is not bytes or bytearray.")
```

Comment:
Since this function should work with or w/o numpy, you could add an test of the behavior w/o numpy... but the numpy route exercises all the lines, so not critical.